src/layers/CustomDisplayObjects/FixedWidthSimpleRope.ts

```diff
@@ -0,0 +1,45 @@
+import { IPoint, Mesh, MeshMaterial, Renderer, RopeGeometry, Texture, WRAP_MODES } from 'pixi.js';
+
+/**
+ * The rope allows you to draw a texture across several points and then manipulate these points
+ * Width of rope is given in constructor
+ */
+export class StaticWidthSimpleRope extends Mesh {
+  /**
+   * re-calculate vertices by rope points each frame
+   * @member {boolean}
+   */
+  public autoUpdate: boolean;
+
+  /**
+   * Note: The wrap mode of the texture is set to REPEAT if `textureScale` is positive.
+   * @param texture - The texture to use on the rope.
+   * @param points - An array of {@link PIXI.Point} objects to construct this rope.
+   * @param {number} textureScale - Optional. Positive values scale rope texture
+   * keeping its aspect ratio. You can reduce alpha channel artifacts by providing a larger texture
+   * and downsampling here. If set to zero, texture will be stretched instead.
+   * @param width - Width of rope
+   */
+  constructor(texture: Texture, points: IPoint[], textureScale = 0, width: number) {
+    const ropeGeometry = new RopeGeometry(width, points, textureScale);
+    const meshMaterial = new MeshMaterial(texture);
+
+    if (textureScale > 0) {
+      // attempt to set UV wrapping, will fail on non-power of two textures
+      texture.baseTexture.wrapMode = WRAP_MODES.REPEAT;
+    }
```

Comment:
Should we include this comment in the contructor docs aswell?

Reply:
It's nothing special with this spesific rope. Its a common thing with using textures in WebGL

Reply:
fixed it